core/meta/inc/TClass.h

```diff
@@ -318,6 +338,14 @@ friend class TStreamerInfo;
 
    TVirtualStreamerInfo     *GetStreamerInfoImpl(Int_t version, Bool_t silent) const;
 
+   mutable TVirtualMutex *fOVRMutex = nullptr;
+   typedef std::multimap<void*, Version_t> RepoCont_t;
+   mutable RepoCont_t fObjectVersionRepository;
+
+   void UnregisterAddressInRepository(const char * /*where*/, void *location, const TClass *what) const;
+   void MoveAddressInRepository(const char * /*where*/, void *oldadd, void *newadd, const TClass *what) const;
+   void RegisterAddressInRepository(const char * /*where*/, void *location, const TClass *what) const;
```

Comment:
Why comment the `where` parameter name?

Reply:
Copy paste from the declaration where the usage of the `where` is commented out as it is "just" for debugging code.